cpp/src/dbscan/dbscan.cuh

```diff
@@ -25,13 +25,12 @@
 
 namespace ML {
 
-using namespace Dbscan;
 // Default max mem set to a reasonable value for a 16gb card.
 static const size_t DEFAULT_MAX_MEM_MBYTES = 13e3;
 
 template <typename Index_ = int>
-Index_ computeBatchCount(size_t &estimated_memory, Index_ n_rows,
-                         size_t max_mbytes_per_batch = 0,
+Index_ computeBatch_size(size_t &estimated_memory, Index_ n_rows,
```

Comment:
should we snake-case the `compute_batch` as well?

Reply:
Oops, it was a side effect of a massive rename of `batchSize` to `batch_size`